### PR TITLE
Add out of stock indication to cart item

### PR DESCRIPTION
**Pull request by yurii-ve**
- add 'out of stock' indication to cart item
- fix cart and cart item boards (pass fullPrice data)

Screenshot 2024-09-22 at 15 01 55

### CODE REVIEW

File: src/components/cart/cart-item/cart-item.module.scss [hunk 1]
```diff
@@ -26,9 +30,10 @@
     font: var(--paragraph2);
 }
 
-.remove {
+.removeButton {
     cursor: pointer;
     margin-left: 6px;
+    color: var(--black);
```
Comment (olehrakwix) [inline src/components/cart/cart-item/cart-item.module.scss:36]: why did you need to add/change this color?

File: src/components/cart/cart-item/cart-item.module.scss [hunk 2]
```diff
@@ -8,6 +8,10 @@
     box-shadow: 0 -1px rgba(40, 43, 45, 0.25);
 }
 
+.isOutOfStock {
```
Comment (olehrakwix) [inline src/components/cart/cart-item/cart-item.module.scss]: probably `outOfStock` is better class name then `isOutOfStock`

File: src/components/cart/cart-item/cart-item.tsx [hunk 3]
```diff
@@ -31,8 +31,18 @@ export const CartItem = ({ cartItem, className, isLast }: CartItemProps) => {
         }
     }
 
+    const isAvailable =
```
Comment (olehrakwix) [inline src/components/cart/cart-item/cart-item.tsx:33]: is this handling variants availability?
I guess PARTIALLY_AVAILABLE comes in case when particular selected variant is in stock?

**Review by olehrakwix — APPROVED** [review]

File: src/components/cart/cart-item/cart-item.tsx (hunk 3 above)
Comment (yurii-ve) [reply src/components/cart/cart-item/cart-item.tsx:33]: From what I understand this is information about variant's availability... 'PARTIALLY_AVAILABLE' is still available to me :)

File: src/components/cart/cart-item/cart-item.module.scss (hunk 1 above)
Comment (yurii-ve) [reply src/components/cart/cart-item/cart-item.module.scss:36]: because root element has class `outOfStock` which set color to grey... this includes close button
